bindings/wysiwyg-ffi/Cargo.toml

```diff
@@ -6,7 +6,7 @@ repository = "https://gitlab.com/andybalaam/wysiwyg-rust"
 description = "Swift and Kotlin bindings for wysiwyg-rust"
 keywords = ["matrix", "chat", "messaging", "composer", "wysiwyg"]
 license = "Apache-2.0"
-name = "wysiwyg-ffi"
+name = "uniffi-wysiwyg-composer"
```

Comment:
I also had to change the name of the lib to not to have to edit the generated bindings for Android every single time new bindings were generated. This might affect iOS @aringenbach.